football/app/models/user.rb

```diff
@@ -2,5 +2,9 @@ class User < ActiveRecord::Base
 	validates :email, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i},
                     presence: true,
                     uniqueness: {message: "alrealy exits"}
+  validates :smartphone, numericality: {only_integer: true}
```

Comment:
add space after `{` and before `}`
`validates :smartphone, numericality: { only_integer: true }`

Please fix other lines.